spyder/plugins/ipythonconsole/utils/manager.py

```diff
@@ -14,10 +14,24 @@
 
 # Third party imports
 from jupyter_client.utils import run_sync
+from qtconsole.client import QtKernelClient, QtZMQSocketChannel
 import psutil
 from qtconsole.manager import QtKernelManager

```

Comment:
```suggestion
import psutil
from qtconsole.client import QtKernelClient, QtZMQSocketChannel
from qtconsole.manager import QtKernelManager
```

Organize alphabetically.